src/records/dnsGeneric.ts

```diff
@@ -0,0 +1,18 @@
+import { Static, Boolean, String, Literal, Record, Union, Partial } from "runtypes";
+
+export const RecordTypesT = Literal("openatts");
+export const AlgorithmT = Union(Literal("dns-did"));
+export const VersionT = String;
+export const PublicKeyT = String;
+
+export const OpenAttestationDnsGenericRecordT = Record({
```

Comment:
its not really generic if its tied to did though?

Reply:
Changed to dnsDid